packages/rum-core/src/domain/view/viewMetrics/trackFirstContentfulPaint.spec.ts

```diff
@@ -40,17 +41,19 @@ describe('trackFirstContentfulPaint', () => {
   })
 
   it('should be discarded if the page is hidden', () => {
+    // stop the previous setup from the beforeEach
+    cleanup()
```

Comment:
**💬 suggestion:** ‏same here, remove `startTrackingFCP` from `beforeEach` and call it in each test case instead